cmd/age-keygen/age-keygen.1

```diff
@@ -0,0 +1,25 @@
+.Dd June 28, 2020
+.Dt AGE-KEYGEN 1
+.Sh NAME
+.Nm age-keygen
+.Nd generate an age secret key
+.Sh SYNOPSIS
+.Nm
+.Op Fl o Ar file
+.Sh DESCRIPTION
+.Nm
+generates an
+.Nm age
+secret key.
+.Pp
+If the
+.Fl o
+flag is not specified the key is output to standard output with the public key
+as a comment.
```

Comment:
This leaves out what the happens with **-o**: What is stored where?

Experienced users will know that **-o** normally means “output file”, sure, but perhaps it'd be useful to specify what it does for all those who aren't familiar with this convention.

Maybe I'm being too pedantic here though.